### PR TITLE
ISSUE_TEMPLATE

### PR DESCRIPTION
**Pull request by gero3**
A initial concept for issue templates

### CODE REVIEW

File: .github/ISSUE_TEMPLATE.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
+##### Description of the problem 
+
+
```
Comment (mrdoob) [inline .github/ISSUE_TEMPLATE.md]: I think I would add `...` here. Markdown seem to be ignoring the line breaks otherwise.

**Comment by mrdoob** [comment]
Yay! Thanks!

**Comment by bhouston** [comment]
BTW you could ask:

```
Before submitting this issue did you:
[ ] Ensure there is not already a Github issue filed on this topic [link]
[ ] Searched for an answer on the StackExchange [link]
```

Something like that.  Although to be honest, the more hoops you place for people submitting issues, the more likely they are to give up before submitting one.

**Comment by bhouston** [comment]
Or you could just have this:
```
Before submitting this issue, please confirm:
[ ] I have read the guide to contributing to Three.JS: [link]
```

**Comment by mrdoob** [comment]
Hmm, I think that's a bit too much for now... It has already reduced the amount of new issues created considerably 😄

**Pull request merged**